src/diffusers/loaders/lora.py

```diff
@@ -522,115 +547,18 @@ def load_lora_into_text_encoder(
                     _pipeline.enable_sequential_cpu_offload()
                 # Unsafe code />
 
-    @classmethod
-    def load_lora_into_transformer(cls, state_dict, network_alphas, transformer, adapter_name=None, _pipeline=None):
```

Comment:
This method is only needed for the `AmusedPipeline`. It's very safe to say that LoRA + Amused isn't used that much. Plus it is convoluting this class unnecessarily. 

To address this, I introduced an `AmusedLoaderMixin` in this PR and performed respective changes in the `train_amused.py` file. This is the only instance where LoRA + Amused is used ever in our codebase.